portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -325,11 +328,103 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-47381. Authorize exernal application."
+	@uitest
```

Comment:
Please group the @uitest label changes into a separate commit with prefix: LRQA-70395